plugins/utils/__init__.py

```diff
@@ -1764,13 +1766,12 @@ def _compute_metadata_inputs(ctx, inputs):
                 default=default_target,
                 view=target_choices,
             )
-
         target = ctx.params.get("target", default_target)
         target_view = _get_target_view(ctx, target)
 
-    if target == "SELECTED_SAMPLES":
+    if target == foo.constants.ViewTarget.SELECTED_SAMPLES:
```

Comment:
Until we migrate these "core" plugins into the `fiftyone` codebase itself, I think it is important that we make every effort to retain backwards compatibility such that users running `fiftyone<1.8` who install this plugin from `fiftyone-plugins:main` branch will get a version of this operator that runs for them.

In this case, that means either:
1. keep using the magic strings
2. pull this new implementation into the `if Version(foc.VERSION) >= Version("1.8.0")` block and keep the current implementation in the `else` block

Reply:
mm right